dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProjectMapper.xml

```diff
@@ -217,4 +212,20 @@
             </foreach>
         </if>
     </select>
+
+    <select id="queryRunningProcessInstanceNumber" resultType="org.apache.dolphinscheduler.dao.model.ProjectInstanceRunningDTO">
+        SELECT COUNT(distinct inst.id) as inst_running_count,def.project_code
+        FROM t_ds_process_definition_log def,
```

Comment:
```suggestion
        FROM t_ds_process_definition def,
```

It's better to query `t_ds_process_definition` instead of `t_ds_process_definition_log`.